Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -489,7 +487,16 @@ - (void)viewDidLoad
 
     [self createConversationScrollButtons];
     [self createHeaderViews];
-    [self createBackButton];
+
+    if (@available(iOS 11, *)) {
```

Comment:
This addresses a longstanding glitch in our interactive transitions on iOS11 vis a vis our custom back button.